components/patient-data-sharing/push-api/src/main/resources/ApplicationResources.properties

```diff
@@ -49,19 +45,17 @@ phenotips.PushPatient.male=Male
 phenotips.PushPatient.unknown=Unknown
 phenotips.PushPatient.negative=NO
 phenotips.PushPatient.oneServerMessage=There is only one configured remote server: 
-phenotips.PushPatient.patientUploadedMessage=This patient was uploaded to
-phenotips.PushPatient.patientUploadedLessThanHour=less than an hour ago
-phenotips.PushPatient.patientUploadedDays=days ago
-phenotips.PushPatient.patientUploadedHours=hours ago
-phenotips.PushPatient.patientUploadedAsID=and was assigned patient ID
+phenotips.PushPatient.patientUploadedLessThanHour=This patient was uploaded to __serverID__ less than an hour ago and was assigned patient ID __patientID__
+phenotips.PushPatient.patientUploadedDays=This patient was uploaded to __serverID__ __daysN__ day(s) ago and was assigned patient ID __patientID__
+phenotips.PushPatient.patientUploadedHours=This patient was uploaded to __serverID__ __hoursN__ hour(s) ago and was assigned patient ID __patientID__
 phenotips.PushPatient.patientUploadedUpdate=If data in the record has changed, push to __serverID__ again to update the record there.
 phenotips.PushPatient.updateRemotePatient=Update remote patient
 phenotips.PushPatient.createRemotePatient=Create new remote patient
 phenotips.PushPatient.loadingServers=Loading server list\u2026
 phenotips.PushPatient.serverDesc=Selected server does not allow updating existing patients, so can only push again to a new patient record.
 phenotips.PushPatient.successfulUpdate=Updated patient successfully.
-phenotips.PushPatient.successfulPush=Pushed patient successfully, ID of the new patient on the remote server is
-phenotips.PushPatient.pushFailedUpdatesDisabled=Unable to update this patient - updates are disbaled on the remote server
+phenotips.PushPatient.successfulPush=Pushed patient successfully, ID of the new patient on the remote server is __patientID__
```

Comment:
Let's change this to something like `The patient was successfully submitted and its identifier on __serverName__ is __patientID__.`

Note that apparently "remote server" doesn't mean anything to our users, so we must be specific about which "server" we're talking about. Apologies about the incomplete description of the issue, but hopefully during review we can catch and fix at least some of the "remote server" occurrences that were missed.